### ISSUE TITLE
Some relabeling rules stopped working v1.108.0

### ISSUE THREAD

**Issue by dguskovv** [VictoriaMetrics/VictoriaMetrics#7865]
### Describe the bug

I use single-node with -influxListenAddr
After upgrade to version 1.108.0 some relabeling rules from -relabelConfig stopped working after victoria-metrics-prod restart.

The rules  is

```
- action:  labeldrop
  if: '{__name__=~"^prometheus_.+"}'
  regex: '^url$'

- source_labels: [__name__]
  regex: '^prometheus_(.+)'
  replacement: $1
  target_label: __name__

```

And this part do not work after start/restart

```
- source_labels: [__name__]
  regex: '^prometheus_(.+)'
  replacement: $1
  target_label: __name__
```

At the debug interface for metric 

```
{__name__="prometheus_node_memory_Cached_bytes",host="test",url="http://127.0.0.1:9100/metrics"}
```

I see all steps, but in fact I get this labelset

```
{__name__="prometheus_node_memory_Cached_bytes",host="test"}
```

### To Reproduce

Run with -relabelConfig mentioned the above

### Version

victoria-metrics-20241213-161529-tags-v1.108.0-0-gb5e4499c29

### Screenshots

![Screenshot 2024-12-17 at 22 59 03](https://github.com/user-attachments/assets/2d0fe4fa-7578-4115-99a2-d379d624333e)

### Used command-line flags

victoria-metrics-prod -storageDataPath=/tmp/test -relabelConfig /tmp/r.conf -influxListenAddr=127.0.0.1:8079 -httpListenAddr=127.0.0.1:6428 -retentionPeriod=12 -inmemoryDataFlushInterval=1h -http.disableResponseCompression

**Comment by f41gh7** [comment]
Hello, thanks for the reporting. An issue was introduced at `v1.108.0` release and only affects `victoria-metrics` single component. 

 It should be fixed at this commit https://github.com/VictoriaMetrics/VictoriaMetrics/pull/7872/commits/e59fffa54a73d2618f3790ba7cb79b3631c06676

 Current workaround - rollback for 1.107.0 or wait for v1.108.1 release

**Comment by f41gh7** [comment]
The bugfix was included into [v1.108.1](https://docs.victoriametrics.com/changelog/#v11081)
